frontend/src/components/pages/CreateReview/editor.css

```diff
@@ -0,0 +1,947 @@
+/*!
```

Comment:
We should maybe move all of our styling to another folder (ie `frontend/styles/**` or something). What do you think?